src/einsteinpy/utils/bl_coord_transforms.py

```diff
@@ -1,244 +0,0 @@
-import warnings
```

Comment:
@OVSofia why are you trying to remove `bl_coord_transforms.py` & `coord_transforms.py` ?
@shreyasbapat did you prompt her to do this??

Reply:
Because of this "+from .bl_coord_transforms import *
No need for these imports" :)

Reply:
He was referring to `symbolic/__init__.py` not `utils/__init__.py`. It goes without saying whatever files are not there inside a folder, should not be imported in `__init__`.
You were not supposed to delete the coordinate transforms from `utils`. Try to read the CI logs and try to understand the shortcomings.

Thanks.

Reply:
https://travis-ci.org/einsteinpy/einsteinpy/builds/546903326?utm_source=github_status&utm_medium=notification
See, it is showing loads of cryptic import errors. Any set up dev environment on your local machine, if you have not done so!

Reply:
Yes, besides of delete the christoffel file I asked him by chat if I should delete files and he said that it will be great if i could also delete bl_coords and coords. That's why I deleted them, but looks like I am understood him wrong. So I put them back, no problem :)

I am working in my local machine with the clone of the fork. 
`(venv) sofia@sofia-OMEN-by-HP-Laptop:~/Escritorio/sofia_fork/einsteinpy$
`
Something else is needed to set it up?

Reply:
No. No! I asked to do so! Sorry that I didn't clarify here!